### ISSUE TITLE
Reg_Cost and Rsv_Cost must be included if ParameterScale=1 even if Reserves=0

### ISSUE THREAD

**Issue by cfe316** [GenXProject/GenX#176]
This is due to 

```
    if setup["ParameterScale"] ==1  # Parameter scaling turned on - adjust values of subset of parameter values
      ...
        inputs_gen["dfGen"][!,:Reg_Cost] = gen_in[!,:Reg_Cost]/ModelScalingFactor # Convert to $ million/GW with objective function in millions
        # Cost of providing spinning reserves
        inputs_gen["dfGen"][!,:Rsv_Cost] = gen_in[!,:Rsv_Cost]/ModelScalingFactor # Convert to $ million/GW with objective function in millions

    end
```

One way or the other these should probably be behind an `if` statement, perhaps inside the existing lines

```
    if setup["Reserves"] >= 1
        # Set for resources with regulation reserve requirements
        inputs_gen["REG"] = gen_in[(gen_in[!,:Reg_Max].>0),:R_ID]
        # Set for resources with spinning reserve requirements
        inputs_gen["RSV"] = gen_in[(gen_in[!,:Rsv_Max].>0),:R_ID]
    end
```

**Comment by cfe316** [comment]
This, and many other issues, could be treated by a refactoring of ModelScalingFactor to always exist, but if ParameterScale=0 just make it `1`. Then we could *always* divide or multiply by it, and fewer branches with duplicate code would be required.

**Comment by cfe316** [comment]
This will be fixed by #366

**Comment by sambuddhac** [comment]
This is now fixed on main
